@re-/assert/test/metaTests/testHelpers.ts

```diff
@@ -0,0 +1,110 @@
+import { strict } from "node:assert"
+import { fromHere, readJson, shell } from "@re-/node"
+import {
+    ExpressionStatement,
+    Project,
+    SourceFile,
+    SyntaxKind,
+    VariableStatement
+} from "ts-morph"
+
+const EXPECTED_RESULTS_JSON_PATH = fromHere("expectedResults.json")
+const PATH_TO_ASSERTIONS_DIR = fromHere(".reassert")
+
+const getExpressionStatements = (
+    functionName: string,
+    varStatements: VariableStatement[]
+) =>
+    varStatements
+        .find((statement) => statement.getText().includes(functionName))
+        ?.getDescendantsOfKind(SyntaxKind.ExpressionStatement)
+
+const getTestComment = (statement: ExpressionStatement, text: string) => {
+    const range = statement.getLeadingCommentRanges()
+    return text
+        .slice(range[0].getPos(), range[0].getEnd())
+        .replace("//", "")
+        .trim()
+}
+export const cleanUpSourceFile = async (sf: SourceFile, template: string) => {
+    sf.replaceWithText(template)
+    await sf.save()
+}
+export const getExpectedResult = (
+    statement: ExpressionStatement,
+    expectedData: Record<string, string>,
+    text: string
+) => expectedData[getTestComment(statement, text)]
+
+export const checkIfTestsContainErrors = (
+    statements: ExpressionStatement[],
+    expectedValues: Record<string, string>,
+    fullText: string
+) => {
+    let hasErrors = false
+    for (const statement of statements) {
+        const expected = getExpectedResult(statement, expectedValues, fullText)
+        try {
+            strict.deepEqual(statement.getText(), expected)
```

Comment:
I'd just remove this whole try block and instead just call `strict.deepEqual` (or perhaps `strict.equal` since you're checking strings) directly after you set expected. If it throws, that's fine and we'll end up getting a clear error in the tests.